### PR TITLE
Intern 4

### PR DESCRIPTION
**Pull request by agarneha1331**

### CODE REVIEW

File: intern.json [hunk 1]
```diff
@@ -0,0 +1,38 @@
+{
+  "coverage": false,
+  "tunnel": "browserstack",
+
+  "tunnelOptions": {
+    "verbose": true,
+    "anyOf": [
+      { "type": "string", "username": "process.env.BROWSERSTACK_USERNAME", "accessKey": "process.env.BROWSERSTACK_ACCESS_KEY" },
```
Comment (karanshah-browserstack) [inline intern.json:8]: Docs impact if we are deleting these files?

File: README.md [hunk 2]
```diff
@@ -8,14 +8,13 @@
...
 * Clone the repo
 * Install dependencies `npm install`
-* Update `*.conf.js` files inside the `conf/` directory with your [BrowserStack Username and Access Key](https://www.browserstack.com/accounts/settings)
+* Update `intern.json` with your [BrowserStack Username and Access Key](https://www.browserstack.com/accounts/settings)
 
 ## Running your tests
-* To run single test, run `npm run single`
-* To run local tests, run `npm run local`
-* To run parallel tests, run `npm run parallel`
+* To run sample test, run `npm run sample-test`
```
Comment (karanshah-browserstack) [inline README.md:14]: Let's keep the scripts and conf.js files consistent with docs and then only rename and modify the files.

Comment (karanshah-browserstack) [reply README.md:14]: Also, let's keep it consistent across all branches.

**Review by karanshah-browserstack — CHANGES_REQUESTED** [review]

File: package.json [hunk 3]
```diff
@@ -4,17 +4,16 @@
   "readme": "INtern Integration with [BrowserStack](https://www.browserstack.com)",
   "description": "Selenium examples for Intern and BrowserStack Automate",
   "scripts": {
-    "test": "npm run single && npm run local && npm run parallel",
-    "single": "./node_modules/.bin/intern-runner config=conf/single.conf.js",
-    "local": "./node_modules/.bin/intern-runner config=conf/local.conf.js",
-    "parallel": "./node_modules/.bin/intern-runner config=conf/parallel.conf.js"
+    "test": "npm run sample-test && npm run local-test",
+    "sample-test": "intern functionalSuites=tests/sample_test.js",
+    "local-test": "intern functionalSuites=tests/local_test.js"
```
Comment (07souravkunda) [inline package.json:9]: change to `sample-local-test`

File: tests/local_test.js [hunk 4]
```diff
@@ -1,18 +1,14 @@
...
-    }
-  });
+registerSuite('BrowserStack Local Testing', {
+  async checkTunnelIsWorking() {
+    await this.remote.get("http://bs-local.com:45454/")
+      .end()
+      .sleep(5000)
+      .getPageTitle()
+      .then(function(title){
+        assert.strictEqual(title, 'BrowserStack Local')
```
Comment (07souravkunda) [inline tests/local_test.js:11]: let's keep includes instead of `strictEqual`.

File: README.md [hunk 5]
```diff
@@ -8,14 +8,13 @@
...
+* To run local tests, run `npm run local-test`
```
Comment (07souravkunda) [inline README.md:15]: `sample-local-test`

File: README.md (hunk 2 above)
Comment (agarneha1331) [reply README.md:14]: This will be finally updated in the master branch along with the docs changes.

File: tests/local_test.js (hunk 4 above)
Comment (agarneha1331) [reply tests/local_test.js:11]: Both for sample-test and sample-local-test?

File: package.json (hunk 3 above)
Comment (agarneha1331) [reply package.json]: done

File: tests/local_test.js (hunk 4 above)
Comment (agarneha1331) [reply tests/local_test.js]: done

File: README.md (hunk 5 above)
Comment (agarneha1331) [reply README.md]: done

File: intern.json [hunk 6]
```diff
@@ -0,0 +1,39 @@
...
+      { "type": "string", "username": "BROWSERSTACK_USERNAME",  "accessKey": "BROWSERSTACK_ACCESS_KEY"}
+    ]
+  },
+
+  "capabilities": {
+    "build": "browserstack build",
+    "browserstack.source": "intern:sample-v4:v1.0",
```
Comment (karanshah-browserstack) [inline intern.json:16]: Why is there a mixture of w3c and non w3c caps? Please make it consitent with selenium version

**Review by karanshah-browserstack — CHANGES_REQUESTED** [review]

**Review by karanshah-browserstack — APPROVED** [review]

**Pull request merged**